proj.sh

```diff
@@ -23,6 +23,10 @@ function build-stack {
   docker compose --env-file $CHOSEN_ENV -f docker-compose.yml -f docker-compose-local.yml build
 }
 
+function build-specific {
+  docker-compose --env-file env-local.env -f docker-compose.yml -f docker-compose-local.yml up -d --no-deps --build $@
```

Comment:
```suggestion
  docker compose --env-file env-local.env -f docker-compose.yml -f docker-compose-local.yml up -d --no-deps --build $@
```

We use `docker compose`  on all other places in this file, and it's also the only alternative that works for on my setup :)